tests/test_driver.py

```diff
@@ -133,3 +136,41 @@ def test_status_not_found(self, driver):
         txid = 'dummy_id'
         with raises(NotFoundError):
             driver.transactions.status(txid)
+
+    def test_prepare(self, driver, alice_pubkey):
+        transaction = driver.transactions.prepare(owners_before=[alice_pubkey])
```

Comment:
the hard-coding was intentional as it provides a view of what the payload looks like